tests/std/tests/P0896R4_ranges_iterator_machinery/test.cpp

```diff
@@ -3249,6 +3249,81 @@ namespace move_iterator_test {
     STATIC_ASSERT(test());
 } // namespace move_iterator_test
 
+namespace counted_iterator_test {
+    using std::bidirectional_iterator_tag, std::default_sentinel_t, std::forward_iterator_tag, std::input_iterator_tag,
+        std::counted_iterator, std::random_access_iterator_tag, std::same_as, std::string, std::three_way_comparable,
+        std::three_way_comparable_with, std::prev, std::next, std::begin, std::end, std::default_sentinel,
+        std::iterator_traits;

```

Comment:
I think we conventionally sort multiple using-declarations lexicographically.

Reply:
Urgh

Reply:
Done